coding-guides/understanding-raw-events.md

```diff
@@ -0,0 +1,111 @@
+# Raw Events
+
+Raw events are used for uncached messages. How would we use raw events? We can use them to check when a message has been edited, reaction was added or removed etc. So, with that being said, let's get started!
+
+> Please keep in mind that we are using discord@11.3.2 for this guide.
+
+So, how would we use said event? We can initialize it like we would with any event!
+
+```js
+const Discord = require('discord.js')
+const client = new Discord.Client()
+client.on('raw', data => {
+  // We will get into more here
+})
+```
+
+This is initializing the `raw` event and we can do things with `data`. "What king of data does `data` provide us?" Below is an example of a message being edited using the data.
+
+```js
+{ t: 'MESSAGE_UPDATE',
+  s: 611,
+  op: 0,
+  d: 
+  // Depending on the message that is being edited, you will get different sets of data.
+   { id: '438455591921254401',
+     embeds: [ [Object] ],
+     channel_id: '265156361791209475',
+     guild_id: '264445053596991498' 
+   }
+}
+```
+> *Not all events can be used with the raw events. Example: MESSAGE DELETE will not work as the message is not cached into the bot, so we cannot get said message.*
```

Comment:
Pretty sure you *do* get message delete except you just don't get the content of the message. To be confirmed.